library/classes/X12Partner.class.php

```diff
@@ -52,6 +61,15 @@ function __construct($id = "", $prefix = "")
         if ($id != "") {
             $this->populate();
         }
+        //***MI2 Add - make sure that we have this column in the table
+        $result = sqlQuery("SHOW COLUMNS FROM `x12_partners` LIKE 'x12_submitter_name'");
```

Comment:
shouldn't add this dynamically, otherwise too tough for project to track. just needs to go in database.sql and the upgrade sql .